### PR TITLE
RHDEVDOCS-5266: Adding a link for steps to add admin username and password…

### PR DESCRIPTION
**Pull request by Dhruv-Soni11**
Version(s):

GitOps1.11+

Issue:

https://issues.redhat.com/browse/RHDEVDOCS-5266

Link to docs preview:

https://71605--docspreview.netlify.app/

Initial peer review: @eromanova97, @Srivaralakshmi 
Peer review: @mburke5678 
SME review: @reginapizza
QE review: @varshab1210

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (reginapizza) [inline modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:11]: You seem to be saying the same sentence twice here?

**Comment by reginapizza** [comment]
Sorry but I don't think this is actually solving the original issue... The request was for instructions on how to find the admin _password_ for additional instances. What you have here is only how to find the _route_ to the additional Argo CD instance dashboard. 

What this should have is steps like this: 
```
4. To log in with your username and password, obtain the password for the Argo CD instance:

   -  In the left panel of the console, use the perspective switcher to switch to the Developer perspective.

   - Use the Project drop-down list and select the openshift-gitops project.

   - Use the left navigation panel to navigate to the Secrets page.

   - Select the openshift-gitops-cluster instance to display the password.

   - Copy the password.

5. Use this password and admin as the username to log in to the Argo CD UI in the new window.
``` 
These were the steps originally stated here: https://docs.openshift.com/gitops/1.11/installing_gitops/installing-openshift-gitops.html#logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account_installing-openshift-gitops 

The only difference for this PR should be that in place of `openshift-gitops` it will be your new Argo CD instance name instead.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:11]: Thanks for pointing this out. Fixed.

**Comment by Srivaralakshmi** [comment]
@Dhruv-Soni11 Please provide the preview link.

File: modules/gitops-argo-cd-installation.adoc
Comment (varshab1210) [inline modules/gitops-argo-cd-installation.adoc:33]: This should be viewed in **Administrator** perspective

File: modules/gitops-argo-cd-installation.adoc
Comment (varshab1210) [inline modules/gitops-argo-cd-installation.adoc:37]: Is this meant for a user defined ArgoCD or the default openshift-gitops instance? In any case I think the better way would be something like this:

```suggestion
.. Use the left navigation panel to navigate to the *Secrets* page.
.. Use the *Project* drop-down list and select the namespace of the Argo CD instance
.. Select the *<Argo_CD_instance_name>-cluster* instance to display the password.
.. Copy the password.
```

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (varshab1210) [inline modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:9]:
```suggestion
{gitops-title} automatically creates a ready-to-use Argo CD instance that is available in the `openshift-gitops` namespace. Optionally, you can create  a new Argo CD instance to manage cluster configurations or deploy applications.
```

File: modules/gitops-argo-cd-installation.adoc
Comment (varshab1210) [inline modules/gitops-argo-cd-installation.adoc:38]: Is there a description provided for accessing the route for ArgoCD server UI? 

Please provide the preview page

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:33]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:37]: As far as I know it is for user defined additional Argo CD instances. Please validate the changes.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:9]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc]: Also, should this term <Argo_CD_instance_name>-cluster be <Argo_CD_instance_name-cluster> instead, @varshab1210??

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:38]: I am unsure about the description for accessing the route @varshab1210. Could you please elaborate a bit more on this? 

Preview page: https://71605--docspreview.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance

File: modules/gitops-argo-cd-installation.adoc
Comment (varshab1210) [reply modules/gitops-argo-cd-installation.adoc]: No, **-cluster** is not a variable

File: modules/gitops-argo-cd-installation.adoc
Comment (varshab1210) [reply modules/gitops-argo-cd-installation.adoc:38]: I see it in the preview page
https://71605--docspreview.netlify.app/openshift-gitops/latest/argocd_instance/setting-up-argocd-instance#gitops-argo-cd-installation_setting-up-argocd-instance
6th step

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Srivaralakshmi) [inline modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:31]:
```suggestion
.. Select the *<argocd_instance_name>-cluster* instance to display the password.
```

Comment (Srivaralakshmi) [inline modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:31]: @Dhruv-Soni11 I hope you have verified and tested the order of these sub-steps and how they work precisely, in the cluster. Please confirm.

Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:31]: Hi @Srivaralakshmi - I just tested the order of these sub-steps and I can confirm that apart from changing the following step from:

Use the left navigation panel to navigate to the *Secrets* page.

to

Use the left navigation panel to navigate to the *Workloads* -> *Secrets* page.

there isn't much to change. Making the change in the content here as well.

Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:31]: Fixed.

Comment (Srivaralakshmi) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:31]: @Dhruv-Soni11 That is great, thanks! Please get the Ack from the SME and QE on these changes.

File: modules/gitops-argo-cd-installation.adoc
Comment (varshab1210) [inline modules/gitops-argo-cd-installation.adoc:36]:
```suggestion
.. Select the *<Argo_CD_instance_name>-cluster* instance to display the password.
```

Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc]: Fixed.

**Comment by varshab1210** [comment]
LGTM, thanks

**Review by reginaelyse — APPROVED** [review]

File: modules/gitops-argo-cd-installation.adoc
Comment (eromanova97) [reply modules/gitops-argo-cd-installation.adoc:26]: We can maybe add the step like this?:
```suggestion
. Go to *Networking* -> *Routes* -> *<instance_name>-server* in the project where the Argo CD instance is installed.

. On the *Details* tab, click the Argo CD web UI link under *Route details* -> *Location*. The Argo CD web UI opens in a separate browser window.
```

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (eromanova97) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:32]: maybe something like this?
```suggestion
.. On the *Details* tab, copy the password under *Data* -> *admin.password*.
```

**Review by eromanova97 — COMMENTED** [review]
Hello @Dhruv-Soni11 I finished my review and I have some comments to consider, please let me know if anything is unclear :) thank you!

File: modules/gitops-argo-cd-installation.adoc
Comment (eromanova97) [reply modules/gitops-argo-cd-installation.adoc:23]: Oh I just noticed that I have been tagged to that Jira yesterday for review as well :smile: Thank you for the clarification!
@Dhruv-Soni11 I apologize if some of my review comments here overlap with the above-linked issue. You can ignore those. Thank you!

Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:23]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:26]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:23]: Fixed, ignoring the changes mentioned in this comment as the other JIRA ticket would be including all the changes.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:29]: Fixed.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:28]: Removing this step as per your suggestion since it is duplicitous as we have already instructed the user to be in the Administrator perspective earlier.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:27]: Fixed.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:32]: Fixed.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:25]: Fixed, PTAL.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:30]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (eromanova97) [reply modules/gitops-argo-cd-installation.adoc]: Hello @Dhruv-Soni11 It might have been easy to overlook, but I wanted to address the missing step I highlight in the last two comments:
```
. On the *Details* tab, click the Argo CD web UI link under *Route details* -> *Location*. The Argo CD web UI opens in a separate browser window.
```
Thank you!

File: modules/gitops-argo-cd-installation.adoc
Comment (eromanova97) [reply modules/gitops-argo-cd-installation.adoc:46]: Hello @Dhruv-Soni11 thank you for adding the missing context! Could you please also address the part I mention in the comment above about the `<user>` parameter? Thank you!

**Copy of the comment:**

This note confuses me a little bit.
Because we explain: `<user> is the default role that you can bind to users and groups...`

We explain this in the OCP docs: 
[oc adm groups new](https://docs.openshift.com/container-platform/4.14/cli_reference/openshift_cli/administrator-cli-commands.html#oc-adm-groups-new)
It explains that <user> variable adds a user to the group, not the role.

Am I missing some understanding here? Because as I understand it, "user" and "role" have different meanings.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:48]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:49]: Fixed.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:32]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:50]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:53]: Fixed.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:29]: Fixed.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc
Comment (Dhruv-Soni11) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc]: I think I have fixed it as per our discussion on Slack. PTAL.

File: installing_gitops/installing-openshift-gitops.adoc
Comment (eromanova97) [inline installing_gitops/installing-openshift-gitops.adoc:43]:
```suggestion
[id="additional-resources_installing-openshift-gitops"]
```

Comment (eromanova97) [reply installing_gitops/installing-openshift-gitops.adoc]: I apologize for suggesting the ID with the spelling error, this is the right ID. Thank you!

Comment (Dhruv-Soni11) [reply installing_gitops/installing-openshift-gitops.adoc]: Fixed.

**Review by eromanova97 — COMMENTED** [review]
Hello @Dhruv-Soni11 I managed to finish the review today to unblock you :) 
I added only 3 small straightforward suggestions, the rest looks good  (excluding that one issue that needs confirmation from engineering) . Thank you!

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:38]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:14]: Fixed.

File: modules/gitops-argo-cd-installation.adoc
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:41]: Fixed.

**Review by eromanova97 — APPROVED** [review]

File: modules/gitops-argo-cd-installation.adoc
Comment (Srivaralakshmi) [reply modules/gitops-argo-cd-installation.adoc]: @eromanova97 Yes, please refer to this slack thread: https://redhat-internal.slack.com/archives/C048WUV584F/p1709305166148199?thread_ts=1709277793.271369&cid=C048WUV584F and read till the end for a complete understanding.  So IMO, we can retain the sentence as is. Thank you!

Comment (eromanova97) [reply modules/gitops-argo-cd-installation.adoc]: Thank you @Dhruv-Soni11 for looking into this and thank you @Srivaralakshmi for your insight! Let's keep the note as it is :smile_cat:

**Comment by Dhruv-Soni11** [comment]
merge-review-needed

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**
